packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -167,6 +168,131 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, named_vip
     return (vip, hosts, cmd, origin_app, proxy_app)
 
 
+def pod_vip_app(network: marathon.Network, host: str, vip: str):
+    ''' Creates a single app in docker container in pod
+    '''
+    container_port = 0
+    if network is not marathon.Network.HOST:
+        container_port = unused_port()
+    test_uuid = uuid.uuid4().hex
+    app = {
+        'id': '/integration-test-{}'.format(test_uuid),
```

Comment:
This is just a question:
Is it possible to set the `id` to a more human-readable string that is representative of the network test combination rather than a random UUID? E.g., `id= test_uuid + "-" + network ?
Since its the app ID that we use in our debug logs below, was hoping that the human readable string might make the logs a lot more intuitive to parse.